pkg/cmd/provisioning/azure/create_managed_identities_test.go

```diff
@@ -118,8 +137,8 @@ func TestCreateManagedIdentities(t *testing.T) {
 				wrapper := mockAzureClientWrapper(mockCtrl)
 				mockGetResourceGroupNotFound(wrapper, testInstallResourceGroupName, testSubscriptionID)
 				mockCreateOrUpdateResourceGroupSuccess(wrapper, testInstallResourceGroupName, testRegionName, testSubscriptionID, resourceTags)
-				mockGetUserAssignedManagedIdentityNotFound(wrapper, testOIDCResourceGroupName, "testinfraname-secretName1-namespace1")
-				mockCreateOrUpdateManagedIdentitySuccess(wrapper, testOIDCResourceGroupName, "testinfraname-secretName1-namespace1", testRegionName, testSubscriptionID, resourceTags)
+				mockGetUserAssignedManagedIdentityNotFound(wrapper, testOIDCResourceGroupName, "testinfraname-namespace1-secretName1")
```

Comment:
Nice catch, I must have reversed these?

Reply:
The [templates](https://github.com/openshift/cloud-credential-operator/pull/556/files#diff-c25bef353d0f432f8ba848ebbe66e75a45d79b2261eb7d4932fa0857a1fa3bb7R80-R81) were in order of 1) `name`, 2) `namespace`, but the template [usage](https://github.com/openshift/cloud-credential-operator/pull/556/files#diff-c25bef353d0f432f8ba848ebbe66e75a45d79b2261eb7d4932fa0857a1fa3bb7R729-R735) was feeding `namespace` before `name`. So it was reversed there, and all of the mock usage was reversed to match.